modules/nsq/src/it/resources/enrich/enrich-nsq.hocon

```diff
@@ -0,0 +1,52 @@
+{
+  "input": {
+    "topic": ${INPUT_TOPIC}
+    "channel": "EnrichNsqChannel"
+    "lookupHost": ${LOOKUP_HOST}
+    "lookupPort": ${LOOKUP_PORT}
+  }
+
+  "output": {
+    "good": {
+      "topic": ${GOOD_OUTPUT_TOPIC}
+      "nsqdHost": ${NSQD_HOST}
+      "nsqdPort": ${NSQD_PORT}
+    }
+
+    "bad": {
+      "topic": ${BAD_OUTPUT_TOPIC}
+      "nsqdHost": ${NSQD_HOST}
+      "nsqdPort": ${NSQD_PORT}
+    }
+  }
+
+  "monitoring": {
+
+    "sentry": {
+      "dsn": ""
+    }
```

Comment:
It's [optional](https://github.com/snowplow/enrich/blob/master/modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/config/io.scala#L399) so you can omit this block